independent-publisher-connectors/NIST National Vulnerability Database/apiDefinition.swagger.json

```diff
@@ -0,0 +1,755 @@
+                                          "type": "array",
+                                          "items": {
+                                            "type": "string"
+                                          },
+                                          "description": "Reference Tags"
+                                        }
+                                      }
+                                    },
+                                    "description": "Reference Data"
+                                  }
+                                },
+                                "description": "References"
+                              },
+                              "description": {
+                                "type": "object",
+                                "properties": {
+                                  "description_data": {
+                                    "type": "array",
+                                    "items": {
+                                      "type": "object",
+                                      "properties": {
+                                        "lang": {
+                                          "type": "string",
+                                          "description": "Vulnerability Description Language"
+                                        },
+                                        "value": {
+                                          "type": "string",
+                                          "description": "Vulnerability Description Value"
+                                        }
+                                      }
+                                    },
+                                    "description": "Vulnerability Description Data"
+                                  }
+                                },
+                                "description": "Vulnerability Description"
+                              }
+                            },
+                            "description": "CVE"
+                          },
+                          "configurations": {
+                            "type": "object",
+                            "properties": {
+                              "CVE_data_version": {
+                                "type": "string",
+                                "description": "CVE Data Version"
+                              },
+                              "nodes": {
+                                "type": "array",
+                                "items": {
+                                  "type": "object",
+                                  "properties": {
+                                    "operator": {
+                                      "type": "string",
+                                      "description": "Logical Operator"
+                                    },
+                                    "children": {
+                                      "type": "array",
+                                      "items": {},

```

Comment:
What is the type of items in the array? All schemas must be well defined.

Reply:
This is a recursive schema where the data type is the same as the parent (node). Hence I left it to implementers here as its impossible to predict the depth